### PR TITLE
Add dependabot config to update GitHub Actions and Dockerfiles.

### PR DESCRIPTION
**Pull request by Ferroin**
SSIA

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
```
Comment (thiagoftsm) [inline .github/dependabot.yml:2]: @Ferroin can we ignore this error?

Comment (Ferroin) [reply .github/dependabot.yml:2]: Theoretically yes, but it’s easier to just fix it...

**Comment by Ferroin** [comment]
@thiagoftsm Just so you know, we’ll probably get a handful of PRs from Dependabot when this gets merged. I’m not sure exactly what it will be trying to update though.

**Review by thiagoftsm — APPROVED** [review]
LGTM!

**Comment by thiagoftsm** [comment]
> @thiagoftsm Just so you know, we’ll probably get a handful of PRs from Dependabot when this gets merged. I’m not sure exactly what it will be trying to update though.

I do not have any PR right now here, but I intend to make another today or tomorrow and we will see what happens. Thanks!

**Pull request merged**